### ISSUE TITLE
Scaffolds without coverage break the script

### ISSUE THREAD

**Issue by jlapaijmans** [jlapaijmans/Consensify#7]
Problem: No output is created by `angsd` for a scaffold without coverage (or only reads that don't pass the filters). `consensify_c` then crashes because it expects a scaffold name that doesn't appear in the .pos or .counts files

Example files attached.
[emptychromosome.zip](https://github.com/jlapaijmans/Consensify/files/14025371/emptychromosome.zip)

Replicate bug behaviour with:
```
./consensify_c -s scaffold_lengths_missingness_emptychroms.txt -p eg_missingness.pos -c eg_missingness.counts -o emptychroms.fasta -min 3 -max 99 -n_matches 2 -n_random_reads 3
```

generates error message:
`ERROR: scaffold in the position and scaffold file do not match, or are out of order
When expecting scaffold2 as read in the positions file, we get scaffold1.5 in the scaffold file`

and truncated output
```
>scaffold1
NGTGANCNACNGAANNAGTTNNCCTNGGNANAACANCGCANTNCTNTTCNNNNNNNNNNN
>scaffold2
```

**Comment by jlapaijmans** [comment]
Default behaviour at the moment is to *not* print empty scaffold, but the flag to turn on empty scaffolds is `-no_empty_scaffold`, which is a bit contradictory (sorry , misunderstanding before). 

So we need either the flag to be renamed to `-empty_scaffold`, OR to change the default behaviour to output empty scaffold and leave the flag as it is. I would lean towards the latter but happy either way, so if its a lot work to change the default behaviour don't bother and we just change the flag.

**Comment by dramanica** [comment]
Implemented as suggested in d909cfa9af5a0ed65746c02fc8dd5cc4391677cc
